src/three-domain-secure/component.test.js

```diff
@@ -1,44 +1,59 @@
 /* @flow */
-import { describe, expect, it, vi } from "vitest";
-import { getSDKToken } from "@paypal/sdk-client/src";
+import { describe, expect, vi } from "vitest";
 
-import { ValidationError } from "../lib";
+import { ThreeDomainSecureComponent } from "./component";
 
-import { getThreeDomainSecure } from "./component";
+const defaultSdkConfig = {
+  sdkToken: "sdk-client-token",
+};
 
-vi.mock("@paypal/sdk-client/src", () => ({
-  getSDKToken: vi.fn(),
-  getLogger: vi.fn(() => ({
+const createThreeDomainSecureComponent = ({
+  sdkConfig = defaultSdkConfig,
+  logger = {
     info: vi.fn().mockReturnThis(),
+    warn: vi.fn().mockReturnThis(),
+    error: vi.fn().mockReturnThis(),
     track: vi.fn().mockReturnThis(),
-    flush: vi.fn().mockReturnThis(),
-  })),
-}));
-vi.mock("../lib", () => ({
-  ValidationError: vi.fn(),
-}));
-describe("getThreeDomainSecure returns ThreeDomainSecureComponent", () => {
```

Comment:
Do we still want to keep this test?

Reply:
I moved this test further down on Line 46 as a general initialization test. Instead of mocking the ValidationError I'm just asserting that it would throw an error. Is this OK or does it change the test? I wasn't entirely sure.

Reply:
I see, yeah that should work too. My idea with that test was to capture `validationError` specifically since it will be one of the trace events in the future. 
This is not a blocker for the PR to be merged